### PR TITLE
Add min-height CSS to print

### PR DESCRIPTION
**Pull request by ronkok**
In release 0.10.0-alpha, we reverted frac-lines to span borders and applied `min-height: 1px` CSS. But that CSS was applied only to screens, not to print. 

This PR removes the media query so that the same `min-height` is applied to both screen and print.

Addresses issue #1593.

### CODE REVIEW

**Comment by ronkok** [comment]
I'm going to withdraw this PR for now. I think we need more information before proceeding. Can the problem reported in issue #1593 be avoided by just using a minimum 10 pt font size? Can it avoided by using Firefox instead of Chrome? Is a `1px` minimum line thickness a net loss for printed material?

Until we know the answers to those questions, we probably should not merge this PR.

**Comment by edemaine** [comment]
While I wish Chrome didn't do any rounding so we could be more TeX-accurate, I think it's reasonable for the printed form to match the screen form in this case, given that the rounding bug sometimes occurs in printed form.

File: src/katex.less [hunk 1]
```diff
@@ -411,6 +404,7 @@
             display: inline-block;
             margin: 0 -0.025em;
             border-right: 0.05em solid;
+            min-width: 1px;
```
Comment (edemaine) [inline src/katex.less]: Now that this line is far away from the rest, maybe it needs a comment too?

Comment (ronkok) [reply src/katex.less]: Done.

**Pull request merged**